src/Umbraco.Tests.AcceptanceTest/cypress/integration/Content/content.ts

```diff
@@ -0,0 +1,418 @@
+/// <reference types="Cypress" />
+import { DocumentTypeBuilder } from 'umbraco-cypress-testhelpers';
+context('Content', () => {
+
+    beforeEach(() => {
+        cy.umbracoLogin(Cypress.env('username'), Cypress.env('password'));
+    });
+
+    it('Copy content', () => {
+        const rootdocTypeName = "Test document type";
+        const childDocTypeName = "Child test document type";
+        const nodeName = "1) Home";
+        const childNodeName = "1) Child";
+        const anotherNodeName = "2) Home";
+
+        const childDocType = new DocumentTypeBuilder()
+                    .withName(childDocTypeName)
+                    .build();
+
+        cy.deleteAllContent();
+        cy.umbracoEnsureDocumentTypeNameNotExists(rootdocTypeName);
+        cy.umbracoEnsureDocumentTypeNameNotExists(childDocTypeName);
+
+        cy.saveDocumentType(childDocType).then((generatedDocType) => 
+        {
+            const rootDocType = new DocumentTypeBuilder()
+                .withName(rootdocTypeName)
+                .withAllowAsRoot(true)
+                .withAllowedContentTypes(generatedDocType["id"])
+                .build();
+
+            cy.saveDocumentType(rootDocType);
+        });
+
+        cy.get('li .umb-tree-root:contains("Content")').should("be.visible").rightclick();
+
+        cy.umbracoContextMenuAction("action-create").click();
+        cy.get('.menu-label').first().click();
+
+        // Type name
+        cy.umbracoEditorHeaderName(nodeName);
+
+        // Save
+        cy.get('.btn-info').click();
+        cy.umbracoSuccessNotification().should('be.visible');
+
+        // Add an item under root node
+        cy.umbracoTreeItem("content", [nodeName]).rightclick();
+        cy.umbracoContextMenuAction("action-create").click();
+        cy.get('.umb-actions').first().click();
+        cy.umbracoEditorHeaderName(childNodeName);
+        cy.get('.btn-info').click();
+
+        // Add second root node
+        cy.get('li .umb-tree-root:contains("Content")').rightclick();
+        cy.umbracoContextMenuAction("action-create").click();
+        cy.get('.menu-label').first().click();
+        cy.umbracoEditorHeaderName(anotherNodeName);
+        cy.get('.btn-info').click();
```

Comment:
I indeed looked for inspiration at the documentTypes tests (specifically the first one), where we use manual clicks instead of the Builder, but I guess I should have checked the second one 😅 where we use an API to setup the data